__tests__/package.spec.js

```diff
@@ -12,6 +12,8 @@
  * the License.
  */
 
+/* eslint-disable jest/no-test-callback */
```

Comment:
well that's embarrassing

might be better to 
```javascript
const path = require('path');
const util = require('util');
const exec = util.promisify(require('child_process').exec);
const packageJson = require('../package.json');
describe('package.json', () => {
  it('should not have peer dep warnings', async () => {
    expect.assertions(2);
    // ex:
    // npm ERR! peer dep missing: eslint@<2.3.0, required by babel-eslint@5.0.4
    const { stdout, stderr } = exec('npm ls', { cwd: path.resolve(__dirname, '../') });
    // if there are no complaints from npm, stdout is null
    expect(stdout || '').not.toMatch('peer dep missing');
    expect(stderr || '').not.toMatch('peer dep missing');
  });
```
than disabling rules in our own repository/project?

Reply:
It's not my code and I didn't immediately know how to make it compliant, so I disabled the rules as a quick solution. If you know how to fix it, please do.

Reply:
Fixed